Sources/HttpParser.swift

```diff
@@ -33,9 +33,15 @@ public class HttpParser {
     }
     
     private func extractQueryParams(_ url: String) -> [(String, String)] {
+        #if compiler(>=5.0)
+        guard let questionMark = url.firstIndex(of: "?") else {
```

Comment:
Should this be renamed to `questionMarkIndex`?

Reply:
This was named before but it makes sense to rename IMO. @alanzeino Can you please update the name?